tests/transforms/test_qcut.py

```diff
@@ -382,6 +382,37 @@ def test_multiple_observables(self):
             assert node_obs.wires == exp_obs.wires
             assert node_obs.obs.name == exp_obs.obs.name
 
+    def test_split_sample_measurement(self):
+        """
+        Test that a circuit with single sample measurement over all wires is
+        correctly converted to a graph with a distinct node for each wire sampled
+        """
+
+        with qml.tape.QuantumTape() as tape:
+            qml.Hadamard(wires=0)
+            qml.CNOT(wires=[0, 1])
+            qml.PauliX(wires=1)
+            qml.WireCut(wires=1)
+            qml.CNOT(wires=[1, 2])
+            qml.sample(wires=[0, 1, 2])
```

Comment:
Can we test similarly for the circuit:
```python
        with qml.tape.QuantumTape() as tape:
            qml.Hadamard(wires=0)
            qml.CNOT(wires=[0, 1])
            qml.PauliX(wires=1)
            qml.WireCut(wires=1)
            qml.CNOT(wires=[1, 2])
            qml.sample(qml.PauliZ(0))
            qml.sample(qml.PauliZ(1))
            qml.sample(qml.PauliZ(2))
```
i.e., that the nodes are
```python
        expected_nodes = [
            qml.Hadamard(wires=[0]),
            qml.CNOT(wires=[0, 1]),
            qml.PauliX(wires=[1]),
            qml.WireCut(wires=[1]),
            qml.CNOT(wires=[1, 2]),
            qml.sample(qml.PauliZ(0)),
            qml.sample(qml.PauliZ(1)),
            qml.sample(qml.PauliZ(2)),
        ]
```
I know we are thinking to restrict to only allow users to specify `qml.sample(wires=[...])` when circuit cutting, but I think the `tape_to_graph` function can already support either choice.

Reply:
Yes you're right, it is supported! Added: 5a4f9e7146c61d1248c0b8dc75310d36226d8988